src/Package/Impl/Publishing/Commands/PreviewPdfCommand.cs

```diff
@@ -24,11 +24,10 @@ internal sealed class PreviewPdfCommand : PreviewCommand {
             if (!await base.CheckPrerequisitesAsync()) {
                 return false;
             }
-            if (!await CheckExistsOnPathAsync("pdflatex.exe")) {
+            if (!await CheckExecutableExistsOnPathAsync("pdflatex")) {
                 var session = _workflowProvider.GetOrCreate().RSession;
                 var message = session.IsRemote ? Resources.Error_PdfLatexMissingRemote : Resources.Error_PdfLatexMissingLocal;
                 await Services.ShowErrorMessageAsync(message);
-                Process.Start("https://miktex.org/2.9/setup");
```

Comment:
We should add platform specific instructions here. Session should have platform info in it.